src/tools/importSmartSheetEvent.test.js

```diff
@@ -82,33 +80,33 @@ describe('Import Smart Sheet Events', () => {
       expect(createdEvents[0].regionId).toEqual(1);
       expect(createdEvents[0].data).toEqual({
         'Sheet Name': 'PD23-24 b. Region 01 PD Plan WITH NCs',
-        'Event ID': 'R01-PD-23-1035',
+        eventId: 'R01-PD-23-1035',
         'Full Event Title': 'R01 Reaching More Children and Families',
-        'Edit Title': 'Reaching More Children and Families',
-        'Event Organizer - Type of Event': 'Regional PD Event (with National Centers)',
-        Audience: 'Recipients',
+        eventName: 'Reaching More Children and Families',
+        eventOrganizer: 'Regional PD Event (with National Centers)',
+        audience: 'Recipients',
         'Event Duration/# NC Days of Support': 'Series',
-        'Reason for Activity': 'Full Enrollment',
-        'Target Population(s)': 'Children/Families affected by traumatic events (select the other reasons for child welfare, disaster, substance use or homelessness)',
-        'Overall Vision/Goal for the PD Event': 'Participants will explore strategies to reach full enrollment including areas of reaching families in greatest need (homelessness/foster care), right sizing-right programming and developing selection criteria',
-        Creator: 'smartsheetevents@ss.com',
+        reasons: ['Full Enrollment'],
+        targetPopulations: ['Children/Families affected by traumatic events (select the other reasons for child welfare, disaster, substance use or homelessness)'],
+        vision: 'Participants will explore strategies to reach full enrollment including areas of reaching families in greatest need (homelessness/foster care), right sizing-right programming and developing selection criteria',
+        creator: 'smartsheetevents@ss.com',
       });
 
       // Assert event 2.
       expect(createdEvents[1].ownerId).toEqual(ownerId);
       expect(createdEvents[1].regionId).toEqual(3);
       expect(createdEvents[1].data).toEqual({
         'Sheet Name': 'PD23-24 b. Region 01 PD Plan WITH NCs',
-        'Event ID': 'R03-PD-23-1037',
+        eventId: 'R03-PD-23-1037',
         'Full Event Title': 'R03 Health Webinar Series: Oral Health and Dental Care from a Regional and State Perspective',
-        'Edit Title': 'Health Webinar Series: Oral Health and Dental Care from a Regional and State Perspective',
-        'Event Organizer - Type of Event': 'Regional PD Event (with National Centers)',
-        Audience: 'Recipients',
+        eventName: 'Health Webinar Series: Oral Health and Dental Care from a Regional and State Perspective',
+        eventOrganizer: 'Regional PD Event (with National Centers)',
+        audience: 'Recipients',
         'Event Duration/# NC Days of Support': 'Series',
-        'Reason for Activity': 'Ongoing Quality Improvement',
-        'Target Population(s)': 'None',
-        'Overall Vision/Goal for the PD Event': 'Oral Health',
-        Creator: 'smartsheetevents@ss.com',
+        reasons: ['Ongoing Quality Improvement'],
+        targetPopulations: ['None'],

```

Comment:
One of the test should exercise the splitArrayTransformer function on either reasons, targetPopulations or both.

Reply:
https://github.com/HHS/Head-Start-TTADP/pull/1559/commits/0c0fe1e9a44f69f0b3fbf7e8d5b907110920674c